[DOC] Fix style issues and add videos to tracing and profiling shared files (#81215)
grafana/grafana: docs/sources/shared/datasources/tempo-traces-to-profiles.md

- Use a simplified configuration with default query, or
- Configure a custom query where you can use a template language to interpolate variables from the trace or span.

To use trace to profiles, navigate to **Explore** and query a trace. Each span now links to your queries. Clicking a link runs the query in a split panel. If tags are configured, Grafana dynamically inserts the span attribute values into the query. The query runs over the time range of the (span start time - 60) to (span end time + 60 seconds).

![Selecting a link in the span queries the profile data source](/media/docs/tempo/profiles/tempo-trace-to-profile.png)

To use trace to profiles, you must have a configured Grafana Pyroscope data source. For more information, refer to the [Grafana Pyroscope data source](/docs/grafana/latest/datasources/grafana-pyroscope/) documentation.

**Embedded flame graphs** are also inserted into each span details section that has a linked profile (requires a configured Grafana Pyroscope data source).
This lets you see resource consumption in a flame graph visualization for each span without having to navigate away from the current view.

> Hover over a particular block in the flame graph to see more details about the resources being consumed.

Author: Lines 33-39 in this file resolve the conflict from the PR. I incorporated the content that was missing from this file.

## Use a basic configuration

To use a simple configuration, follow these steps:

1. Select a Pyroscope data source from the **Data source** drop-down.
1. Optional: Choose any tags to use in the query. If left blank, the default values of `service.name` and `service.namespace` are used.

>    The tags you configure must be present in the spans attributes or resources for a trace to profiles span link to appear.

Reviewer: I don't know the terminology here... is `a trace to profiles span link` a name for the kind of link? or is a word missing here?

Author: So the feature name is "trace to profiles", which makes me wonder if it should be "traces-to-profiles span link" since traces-to-spans is functioning as an adjective.

Reviewer: The hyphens def help :-)

Author: Okay!
```suggestion
   The tags you configure must be present in the spans attributes or resources for a trace-to-profiles span link to appear.
```

To use a custom query with the configuration, follow these steps:

1. Select a Pyroscope data source from the **Data source** drop-down.
1. Optional: Choose any tags to use in the query. If left blank, the default values of `service.name` and `service.namespace` are used.

>    These tags can be used in the custom query with `${__tags}` variable. This variable interpolates the mapped tags as list in an appropriate syntax for the data source and only includes the tags that were present in the span omitting those that weren’t present. You can optionally configure a new name for the tag. This is useful in cases where the tag has dots in the name and the target data source doesn't allow using dots in labels. For example, you can remap `service.name` to `service_name` in such a case. If you don’t map any tags here, you can still use any tag in the query like this `method="${__span.tags.method}"`.

Author: Thank you for removing future tense. Slight wording suggestion to tighten the text a little: 
```suggestion
   These tags can be used in the custom query with `${__tags}` variable. This variable interpolates the mapped tags as list in an appropriate syntax for the data source. Only the tags that were present in the span are included; tags that aren't present are omitted. You can also configure a new name for the tag. This is useful in cases where the tag has dots in the name and the target data source doesn't allow using dots in labels. For example, you can remap `service.name` to `service_name`. If you don’t map any tags here, you can still use any tag in the query, for example: `method="${__span.tags.method}"`.
```
